### DISCUSSION TITLE
Could we deploy BanyanDB 6.0 using helm?

### DISCUSSION THREAD

**Discussion by xiangzirui** [apache/skywalking#12215] [Q&A]
I want to deploy banyanDB on kubernetes because of the skywalking 10.0.0
but I check to see that its latest version is skywalking-banyandb-helm v0.1.0.
Will it be updated with banyan 6.0?

**Comment by wu-sheng** [comment]
skywalking-banyandb-helm v0.2.0 is coming. fyi @hanahmily
